### DISCUSSION TITLE
Trying to login with Twitter / Discord / Google is impossible on the mobile (react web app) on android

### DISCUSSION THREAD

**Discussion by duball69** [WalletConnect/walletconnect-monorepo#5279] [Developer Support]
Hey, I have been trying to use WalletConnect to have Email & Socials on my dapp. But can't seem to be able to login with Twitter / Discord or Google on the mobile. On the last step of the connection there is a screen that says: Sucessfully logged in. Return to the application, do not close this tab". Issue is you can't go back to the application without closing that tab, since you are still on the app (React web app) page.

This happens using Android with a react web app.

Thank you 

![image](https://github.com/user-attachments/assets/750e80cf-b53d-4c0e-89f6-26513a3e4791)

**Comment by rtomas** [comment]
Cc @ignaciosantise

**Reply by ignaciosantise** [reply]
Seems a web sdk issue. Cc @svenvoskamp
